osu.Game/Rulesets/Mods/ModEasy.cs

```diff
@@ -21,7 +21,7 @@ public abstract class ModEasy : Mod, IApplicableToDifficulty, IApplicableFailOve
         public override bool Ranked => true;
         public override Type[] IncompatibleMods => new[] { typeof(ModHardRock), typeof(ModDifficultyAdjust) };
 
-        [SettingSource("Extra Lives", "Number of extra lives")]
+        [SettingSource("Extra Lives", "Number of extra lives", "{0} lives")]
```

Comment:
This one is actually tricky, as it's incorrect for a value of 1... You probably want to actually override here and use `Humanizer.ToQuantity` to pluralise properly.